src/com/palmergames/bukkit/towny/db/TownySQLSource.java

```diff
@@ -1899,6 +1890,7 @@ private boolean loadJail(ResultSet rs) {
 				TownyMessaging.sendErrorMsg("SQL: A jail was not registered properly on load!");
 				return true;
 			}
+			uuid = jail.getUUID().toString();
```

Comment:
Doesn't seem to be used

Reply:
This one is used way down lower in the catches.